src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -279,8 +278,6 @@ const UsersFragment = ({
     );
   };
 
-  if (!isReady) return <></>;
-
   if (isLoadingDiscoveryUser)
     return (
       <View style={styles.fragmentContainer}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition for rendering the loading state has been changed from `!isReady` to `isLoadingDiscoveryUser`. Ensure that this change in logic is intended and correctly reflects the desired behavior of the application.
```diff
- if (!isReady) return <></>;
+ if (isLoadingDiscoveryUser)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->